### PR TITLE
chore: update help doc with OCI layout support

### LINKED ISSUE #904: add OCI layout support in command help doc
### What happened in your environment?

Below command do support operating an OCI layout folder, but the short and long description only mentioned the support for remote registry
- `oras discover`
- `oras pull`
- `oras push`
- `oras tag`
- `oras blob`
- `oras blob push`
- `oras blob fetch`
- `oras manifest`
- `oras manifest push`
- `oras manifest fetch`
- `oras manifest fetch-config`

### What did you expect to happen?

Short and long description of below commands should mention OCI layout support. 
- `oras discover`
- `oras pull`
- `oras push`
- `oras tag`
- `oras blob`
- `oras blob push`
- `oras blob fetch`
- `oras manifest`
- `oras manifest push`
- `oras manifest fetch`
- `oras manifest fetch-config`

### How can we reproduce it?

Run `<command> -h`

### What is the version of your ORAS CLI?

1.0.0

### What is your OS environment?

20.04

### Are you willing to submit PRs to fix it?

- [ ] Yes, I am willing to fix it.

### PR DESCRIPTION
**Pull request by qweeah**
**What this PR does / why we need it**:
This PR mentioned OCI layout support in related command help doc.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #904

**Please check the following list**:
- [ ]  Does the affected code have corresponding tests, e.g. unit test, E2E test?
- [ ]  Does this change require a documentation update?
- [ ]  Does this introduce breaking changes that would require an announcement or bumping the major version?
- [ ]  Do all new files have an appropriate license header?

### CODE REVIEW

**Comment by qweeah** [comment]
@FeynmanZhou Can you help review the wording in the updated help doc, thanks

File: cmd/oras/root/pull.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ func pullCmd() *cobra.Command {
 		Short: "Pull files from remote registry",
 		Long: `Pull files from remote registry
 
-Example - Pull artifact files from a registry:
+Example - Pull artifact files from a registry or an OCI layout folder:
```
Comment (shizhMSFT) [inline cmd/oras/root/pull.go:57]: This example is to pull artifact files from a registry. It does not reflect the OCI layout folder.

Comment (qweeah) [reply cmd/oras/root/pull.go:57]: yeah, accepted the wrong replacement, will correct it.

File: cmd/oras/root/blob/fetch.go [hunk 2]
```diff
@@ -45,8 +45,8 @@ func fetchCmd() *cobra.Command {
 	var opts fetchBlobOptions
 	cmd := &cobra.Command{
 		Use:   "fetch [flags] {--output <file> | --descriptor} <name>@<digest>",
-		Short: "Fetch a blob from a remote registry",
-		Long: `Fetch a blob from a remote registry
+		Short: "Fetch a blob from a registry or an OCI layout folder",
```
Comment (FeynmanZhou) [inline cmd/oras/root/blob/fetch.go:48]:
```suggestion
		Short: "Fetch a blob from a registry or an OCI image layout folder",
```

Comment (FeynmanZhou) [reply cmd/oras/root/blob/fetch.go:48]: We will create a glossary document to explain every terminology. I would suggest using the OCI image layout here by following the [official term](https://github.com/opencontainers/image-spec/blob/main/image-layout.md).

**Review by FeynmanZhou — COMMENTED** [review]
Just a tiny suggestion.

**Comment by qweeah** [comment]
> Just a tiny suggestion.

Resolved.  @FeynmanZhou  Worth mentioning that the naming of the flag is still using OCI layout, e.g. `--from-oci-layout`

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**